pom.xml

```diff
@@ -153,12 +153,6 @@
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
-            <artifactId>pipeline-stage-step</artifactId>
```

Comment:
Ah, okay that makes it clearer.  Okay, retracting my bug and granting the 🐝 instead!